module-2/backend/week_three.md

```diff
@@ -9,15 +9,65 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 
 ### Questions
 
-1. What is the entry at the command line to create a new rails app?
-2. What do Models generally inherit from in rails?
-3. What do Controllers generally inherit from in a rails project?
-4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
-5. What rake task is useful when looking at routes, and what information does it give you?
-6. What is an example of a route helper? When would you use them?
-7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
-8. What are strong params and why are the necessary?
-9. What role does `form_for` play in helping us create our forms?
-10. How does `form_for` know where to submit the user's input?
-11. Create a form using a `form_for` helper to create a new `Horse`. 
-12. Why do we want to validate our models?
+**1. What is the entry at the command line to create a new rails app?**
+
+rails new <project_name> --T --database=postgresql --skip-spring --skip-turbolinks
+
+**2. What do Models generally inherit from in rails?**
+```ruby
+ApplicationRecord
+```
+**3. What do Controllers generally inherit from in a rails project?**
+```ruby
+ApplicationController
+```
+**4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?**
+```ruby
+resources :horses, only [:show]
+```
+**5. What rake task is useful when looking at routes, and what information does it give you?**
+
+rake routes. It gives you a list of the available routes with their URI, action, and prefix.
+
+**6. What is an example of a route helper? When would you use them?**
+```ruby
+horses_path(horses)
+```
+Anytime you are routing through rails, it is a simplified nomenclature for the path.
+
+**7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?**
+
+
+
+**8. What are strong params and why are the necessary?**
+
+Prevents data you do not want from being passed into your controller.
+
```

Comment:
data will still come to the controller but this will prevent data from being stored in your database